storage/iscsi-tools/open-iscsi/pkg.yaml

```diff
@@ -51,7 +51,7 @@ steps:
           -Dno_systemd=true \
           -Dprefix=/usr/local \
           -Discsi_sbindir=/usr/local/sbin \
-          -Drulesdir=/usr/local/etc/udev/rules.d \
+          -Drulesdir=/usr/lib/udev/rules.d \
```

Comment:
I this this is the default setting in most packages, if so I guess we can drop the config even

Reply:
Should I perhaps just remove this config line? And the similar path in nut-client?

Reply:
yes, let's do that as long as it installs the default rules at the right location

Reply:
we also need to bump extensions-validator

Reply:
open-iscsi defaults to `RULESDIR ?= $(etcdir)/udev/rules.d`, so for it I must keep the line

Reply:
And well, NUT also seems to default to etc